pkg/storagecapabilities/storagecapabilities.go

```diff
@@ -262,6 +267,17 @@ var storageClassToProvisionerKeyMapper = map[string]func(sc *storagev1.StorageCl
 		}
 		return "UNKNOWN"
 	},
+	"infinibox-csi-driver": func(sc *storagev1.StorageClass) string {
```

Comment:
@awels wasn't there a guideline to structuring the provisioner string? this one seems off but definitely the correct one

Reply:
So in general if there is a `/` in the provisioner string, it is in an in tree driver. And csi drivers will have all `.` in them. It is weird to see a csi driver with a `/` in there. But there are no rules against it I guess.

Reply:
I guess in this case the `/` is differentiating the type of back-end used for the storage class.